build.rs

```diff
@@ -0,0 +1,187 @@
+use bzip2::read::BzDecoder;
+use std::{
+    env,
+    fs::{create_dir_all, write as fs_write, File},
+    path::{Path, PathBuf},
+    process::Command,
+};
+use tar::Archive;
+use zip::ZipArchive;
+
+const TESTGEN_HS_PATH: &str = "TESTGEN_HS_PATH";
+
+fn main() {
+    if env::var(TESTGEN_HS_PATH).is_ok() {
+        println!(
+            "Environment variable {} is set. Exiting the build script.",
+            TESTGEN_HS_PATH
+        );
+        return;
+    }
+
+    // Tell Cargo to rerun the build script if the build script itself changes.
+    println!("cargo:rerun-if-changed=build.rs");
+
+    let testgen_lib_version = "10.1.2.1";
```

Comment:
This is 10.1.4.0 on `master` for the 10.1.4 `cardano-node`. :)

I wonder, in principle we _could_ read it from the `flake.lock` to define this version in a single place only, something like (Bash):

```
❯ jq <flake.lock -r '.nodes["testgen-hs"].original.ref'
10.1.4.0
```

But maybe it's too much, and it's enough to write a comment in `flake.nix` to also remember to update it here?

Reply:
Moved to:
* #115.